Matlab/Matlab.sublime-syntax

```diff
@@ -1,36 +1,61 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
+# https://www.sublimetext.com/docs/syntax.html
+# https://www.mathworks.com/help/matlab/index.html
 name: MATLAB
+scope: source.matlab
```

Comment:
I've updated the syntax in this PR to use version 2 now.

This could indeed be useful to allow potential inheritance for Octave, or if someone e.g. wants to add highlighting for Matlab toolbox functions.